src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
You should move this one to the 'normalize' file and use it on all elements 
```suggestion
* {
  box-sizing: border-box;
  }
```

Reply:
Done!